### DISCUSSION TITLE
Native Stack vs Stack - difference in 'back' behaviour in nested navigators

### DISCUSSION THREAD

**Discussion by ChrisEdson** [react-navigation/react-navigation#10100] [Q&A]
I've been upgrading my navigators in my app from `@react-navigation/stack` to `@react-navigation/native-stack`:

I currently nest Stack Navigators in order to hide the bottom tab bar when navigating to a 'Chat' component of my app.

When upgrading from `stack` navigators to `native-stack` navigators - the back button no longer appears on the header of the nested screen.

### With Stack Navigator

https://user-images.githubusercontent.com/3041427/140162919-2485da12-71e3-4579-a683-4c16cfa5d67f.mp4

### With Native Stack Navigator

https://user-images.githubusercontent.com/3041427/140162941-19391d1f-745d-47fd-bfed-95ef8538424d.mp4

Does anyone have any tips to get the back button to come back on the Native Stack navigator implementation?

**Comment by ChrisEdson** [comment]
A very gentle bump on this if anyone has any thoughts 🙏

**Comment by kickbk** [comment]
I got the exact same question. @satya164 is there a way we can keep the same back button behavior as with `@react-navigation/stack`?
I sort of guess that because we move to a different stack, it carries its own navigation history, and the screen we navigated from to the nested stack is not part of the stack, so there's no back button?
What do we do about this?
